examples/v1beta1/pipeline/realtimeimageclassfication/pipeline/Train/train.py

```diff
@@ -0,0 +1,171 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import click
+import os
+import logging
+import shutil
+import ast
+import numpy as np
+import pathlib
+import json
+import tensorflow as tf
+from storage import Storage
+from tensorflow import keras
+from tensorflow.keras import layers
+from tensorflow.keras.models import Sequential
+from tensorflow.python.keras.backend import dropout
+
+
+def build_model(learning_rate,data_augmentation,opt):
```

Comment:
Do you need separate scripts for Train and Tune model ?
You might just re-use the same scripts for tuning and training.

Reply:
First tune is for hyperparameter tuning and the best model goes for final training where we saving the model artifact's in GCP bucket.

Reply:
But what is the differences between teaching model in Tuning step and in Training step ?

If the training process is the same, we might use the command line argument to tell the script if we need to save model artifacts in GCP bucket.

What do you think @aniruddhachoudhury ?

Reply:
Ya we can do that also. I will check that once will let you know the changes.

Reply:
Lets keep it separate so that others can re-use the framework for different use-case

Reply:
What use-cases do we have when training and tuning process is different ?